Add link to EUI and rework home page

florent-leborgne opened the pull request:
This PR changes a few things around the home page:

- Move the home page out of any category 
<img width="259" alt="image" src="https://user-images.githubusercontent.com/10208282/166262419-6e9361f6-98a8-42c5-a4bd-466406329031.png">

- Update the content of the home page 
<img width="1895" alt="image" src="https://user-images.githubusercontent.com/10208282/166262350-b8502036-de0a-47c5-a558-fa4ffa327c6c.png">

- Add a link to the EUI docs 
<img width="414" alt="image" src="https://user-images.githubusercontent.com/10208282/165926164-d678d8bc-cc07-47d1-984e-34b15387e5a4.png">

gchaps commented on docs/home.mdx:
```diff
@@ -1,22 +1,77 @@
+    [
+      {
+        "title": "UI Copy placeholder",
+        "description": "Terminology, resources, and tips for writing in and about the user interface.",
```
If I'm thinking of the right doc, then maybe:

```suggestion
        "description": "Terminology, resources, and tips for writing docs about the user interface.",
```

florent-leborgne replied:
Now that István has included the page, I've used the description of that page directly.

gchaps commented on docs/home.mdx:
```diff
@@ -1,22 +1,77 @@
+        "pageId": "inProdHelp"
+      },
+       {
+        "title": "Writing for specific components",
```
Move this one to the first position.  How about listing what some of the components might be:

>**UI copy**
>Guidelines and examples for writing UI copy for for buttons, menus, error messages, and more.

gchaps commented on docs/home.mdx:
```diff
@@ -1,22 +1,77 @@
+       {
+        "title": "Writing for specific components",
+        "description": "Find writing guidelines for specific UI components directly in the EUI docs.",
+        "href": "https://eui.elastic.co",
```
Link to this page:  https://eui.elastic.co/#/guidelines/writing/guidelines

gchaps replied:
<img width="227" alt="Screen Shot 2022-04-29 at 8 13 01 AM" src="https://user-images.githubusercontent.com/33642766/165973185-83045bf3-141b-4ef0-a1c2-e7a5f92a7e52.png">

Context help in Kibana > In-product help

gchaps commented on docs/home.mdx:
```diff
@@ -1,22 +1,77 @@
+  }
+/>
 
+<DocRelatedArticles
```
How about moving the copy section above the docs section?

florent-leborgne replied:
Good idea, doing it in the TOC as well

gchaps commented on docs/home.mdx:
```diff
@@ -1,22 +1,77 @@
+  items={
+    [
+      {
+        "title": "Docs placeholder",
```
We also have guidelines for deprecation messages that we could add here.

florent-leborgne replied:
👍 István added them with his last merge, I'll pull the changes to this branch

florent-leborgne commented on docs/home.mdx:
```diff
@@ -1,22 +1,77 @@
+  items={
+    [
+      {
+        "title": "Docs placeholder",
```
+1, placeholders were while waiting for additional pages to be pushed, I'll update with the pages we've moved to this repo already

florent-leborgne commented on docs/home.mdx:
```diff
@@ -1,22 +1,77 @@
 ---
 
-Welcome, Documentarian!
+Welcome, content designer!
```
I'm trying with just bold to start with and avoid having too many titles. In fact, I'd like to shorten the intro text so the link sections are more visible.
Another option would be to use the title of the page, currently *Elastic Content Design Home* and tweak it a little.

florent-leborgne commented on docs/home.mdx:
```diff
@@ -1,22 +1,77 @@
+  items={
+    [
+      {
+        "title": "UI Copy placeholder",
```
I had added placeholders while waiting for new pages to be merged. I'm updating those in the next commit.

florent-leborgne commented:
@gchaps & @szabosteve, I've updated the PR based on the latest PRs merged by István and the feedback you provided, plus a few additional changes:

- "Writing about the UI" -> I've moved this section into the docs section

Some of the things that are still up in the air:

- The docs and UI copy sections start with pages that are for now empty, do we want to remove those or do you have plans for them @szabosteve? 
- We probably can shorten the intro of the home, or at least make a few tweaks so that readers's eyes get more quickly to the different link sections. 
- To be able to go live and publish to the internal next docs, we'll probably have to review the IDs of all our files and prefix them with something proper to our repo to avoid any duplicate in the wordlake-dev repo, that'll just grow and grow as teams get onboarded to the new system. maybe something as simple as `cd` (for content design)? 
- The tile that links to the EUI guidelines works, but it throws a non-blocking error on my local dev environment, I' will check this with the next-docs team.

gchaps commented on docs/home.mdx:
```diff
@@ -1,23 +1,81 @@
 
-This guide provides editorial guidelines for writing clear and consistent 
-documentation and UI copy for Elastic products.
+No matter if you are a tech writer, designer, developer, product manager, support
```
To shorten the intro, you could remove this paragraph.

florent-leborgne commented:
Oops I pushed with a wrong account

florent-leborgne commented:
I'm closing this PR and will open a clean one
